java/arcs/core/entity/EntityBase.kt

```diff
@@ -181,6 +181,14 @@ open class EntityBase(
     }
 }
 
+class EntityBaseSpec(
+    override val SCHEMA: Schema
+) : EntitySpec<EntityBase> {
+    init { SchemaRegistry.register(this) }
+    override fun deserialize(data: RawEntity): EntityBase =
+        EntityBase("EntityBase", SCHEMA).apply { deserialize(data) }
+}
+
```

Comment:
Shane you may want to sync with Piotr/Alex, because I told them to make something called a 'DynamicEntity' (as opposed to static subclasses) that extends entitybase and I think pretty much does what you need.  I think this is fine for now, and their stuff can be dropped in later, but if you have any additional requirements in the API it might be good to throw them in Alex's doc.

Reply:
As we discussed in the BAR; DynamicEntity adds nothing over BaseEntity and would just represent additional complexity.